ts/screens/authentication/IdpSelectionScreen.tsx

```diff
@@ -133,6 +134,7 @@ class IdpSelectionScreen extends React.PureComponent<Props, State> {
       return;
     }
     this.props.setSelectedIdp(idp);
+    instabugLog(`IDP selected: ${idp.id}`, TypeLogs.DEBUG, "login");
```

Comment:
I guess we already log this information ([set it as attribute](https://github.com/pagopa/io-app/blob/be27ebbf8e1f0e5c34bd94d4f9e50984b665d3ec/ts/boot/configureInstabug.ts#L95))

![Schermata 2020-07-17 alle 11 06 03](https://user-images.githubusercontent.com/822471/87769235-a3723880-c81d-11ea-88ae-28d6531211ec.png)

Reply:
this log has been added in order to evaluate how long it has been since the user started the login and received any error message.